developer_guides/unit_tests.rst

```diff
@@ -80,10 +80,47 @@ Preparing cmocka package
 
 #. Use the target location for cmocka files when invoking *configure* script:
 
-   .. code-block:: bash
+.. code-block:: bash
 
       ./configure --with-cmocka-prefix=/home/<you>/cminstall ...
 
+Wrapping objects for unit tests
+******************************

```

Comment:
@deb-intel this underline is too short

Reply:
fixed in #29 https://github.com/thesofproject/sof-docs/pull/29/commits/4207d17ac70bb98c6d9c1f9faa6f62774d58b4e4